docs/task_information.rst

```diff
@@ -51,6 +55,85 @@ An example output is as follows:
        └─-(COMPLETE) TaskA[2549878535c070fb6c3cd4061bdbbcff](parameter={'workspace_directory': './resources/', 'local_temporary_directory': './resources/tmp/', 'param': 'called by TaskB'}, output=['./resources/output_of_task_a_2549878535c070fb6c3cd4061bdbbcff.pkl'], time=0.0009829998016357422s, task_log={})
 
 
+On Python
+~~~~~~~~~
+
+It use :func:`~gokart.info.make_tree_info` in the following:
+
+
+.. code:: python
+
+    import luigi
+    import gokart
+
+    class TaskA(gokart.TaskOnKart):
+        param = luigi.Parameter()
+        def run(self):
+            self.dump(f'{self.param}')
+
+    class TaskB(gokart.TaskOnKart):
+        task = gokart.TaskInstanceParameter()
+        def run(self):
+            task = self.load('task')
+            self.dump(task + ' taskB')
+
+    class TaskC(gokart.TaskOnKart):
+        task = gokart.TaskInstanceParameter()
+        def run(self):
+            task = self.load('task')
+            self.dump(task + ' taskC')
+
+    class TaskD(gokart.TaskOnKart):
+        task1 = gokart.TaskInstanceParameter()
+        task2 = gokart.TaskInstanceParameter()
+        def run(self):
+            task = [self.load('task1'), self.load('task2')]
+            self.dump(','.join(task))
+
+
+The more dependencies you have, the harder it is to see the tree.
+
+
+.. code:: python
+
+    task = TaskD(
+        task1=TaskD(
+            task1=TaskD(task1=TaskC(task=TaskA(param='foo')), task2=TaskC(task=TaskB(task=TaskA(param='bar')))),  # same task
+            task2=TaskD(task1=TaskC(task=TaskA(param='foo')), task2=TaskC(task=TaskB(task=TaskA(param='bar'))))   # same task
+        ),
+        task2=TaskD(
+            task1=TaskD(task1=TaskC(task=TaskA(param='foo')), task2=TaskC(task=TaskB(task=TaskA(param='bar')))),  # same task
+            task2=TaskD(task1=TaskC(task=TaskA(param='foo')), task2=TaskC(task=TaskB(task=TaskA(param='bar'))))   # same task
+        )
+    )
+    print(gokart.make_tree_info(task))
+
+
+.. code:: sh
+
+    └─-(PENDING) TaskD[187ff82158671283e127e2e1f7c9c095]
+        |--(PENDING) TaskD[ca9e943ce049e992b371898c0578784e]    # duplicated TaskD
+        |  |--(PENDING) TaskD[1cc9f9fc54a56614f3adef74398684f4]    # duplicated TaskD
+        |  |  |--(PENDING) TaskC[dce3d8e7acaf1bb9731fb4f2ae94e473]
+        |  |  |  └─-(PENDING) TaskA[be65508b556dd3752359b4246791413d]
+        |  |  └─-(PENDING) TaskC[de39593d31490aba3cdca3c650432504]
+        |  |     └─-(PENDING) TaskB[bc2f7d6cdd6521cc116c35f0f144eed3]
+        |  |        └─-(PENDING) TaskA[5a824f7d232eb69d46f0ac6bbd93b565]
+        |  └─-(PENDING) TaskD[1cc9f9fc54a56614f3adef74398684f4]
+        |     └─- ...
+        └─-(PENDING) TaskD[ca9e943ce049e992b371898c0578784e]
+            └─- ...
+
+
+This has been omitted.
```

Comment:
It is ambiguous that what `This` specified, IMO.

If you mean `This` equals `duplicated tasks`, then this sentence should be as follows;

```
In task dependency tree output by `make_tree_info`, the sub-trees already shown in above will be omitted.
```

Reply:
Looks good. thx.